com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/HDRaytracingReflection.cs

```diff
@@ -520,6 +510,97 @@ static void RenderQualityRayTracedReflections(CommandBuffer cmd, RTRQualityRende
             CoreUtils.SetKeyword(cmd, "MULTI_BOUNCE_INDIRECT", false);
         }
 
+        #endregion
+
+        #region Legacy Pipeline
```

Comment:
What is "Legacy pipeline"?

Reply:
Non-render graph

Reply:
this pr took too much time to get merged. @JulienIgnace-Unity removed this part of the code i think